controllers/imageclusterinstall_controller.go

```diff
@@ -261,6 +261,10 @@ func (r *ImageClusterInstallReconciler) Reconcile(ctx context.Context, req ctrl.
 		}
 		return ctrl.Result{}, err
 	}
+	if updateErr := r.setRequirementsMetCondition(ctx, ici, corev1.ConditionTrue, v1alpha1.HostValidationSucceeded, v1alpha1.HostValidationsOKMsg); updateErr != nil {
```

Comment:
I think this should still be after the call to `setHostConfiguredCondition` in the error case for `updateBMHProvisioningState`, right?

Reply:
Hmm, I think I rather fix `updateBMHProvisioningState` the only error case there is failure to path the BMH with an annotation, that isn't related to the requirementsMet condition.